test/albums.test.js

```diff
@@ -120,3 +128,92 @@ describe('user Albums List', () => {
       );
   });
 });
+
+describe('list of user albums photos', () => {
+  it('should list the photos of the albums of a user', done => {
+    factoryCreate({
+      name: 'yesica',
+      lastName: 'nava',
+      email: 'yesica@wolox.co',
+      password: 'shdfgs345',
+      administrator: true
+    })
+      .then(() => {
+        factoryCreateAlbums();
+      })
+      .then(() =>
+        request(app)
+          .post('/users/sessions')
+          .send({ email: 'yesica@wolox.co', password: 'shdfgs345' })
+          .set('Accept', 'application/json')
+          .then(response => {
+            request(app)
+              .get('/users/albums/1/photos')
+              .set({ Accept: 'application/json', Authorization: response.body.token })
+              .then(result => {
+                expect(result.statusCode).toBe(200);
+                expect(result.body).toEqual(albumPhotos);
+                done();
+              });
+          })
+      );
+  });
+
+  it('should not list the photos of the albums of a user', done => {
```

Comment:
the test is good but the description could be better, something like 'should not list the photos of the albums of another user'

Reply:
perfect!